src/components/oculus-touch-controls.js

```diff
@@ -50,8 +51,7 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
       button5: 'surface'
     },
     'right': {
-      axis0: 'thumbstick',
-      axis1: 'thumbstick',
+      axes: {'thumbstick': [0, 1]},
```

Comment:
so like axes, buttons would be a keyed map of either singletons or arrays (if aliased)

buttons: { thumbstick: 0, ... }

more readable IMO